runtime/lua/vim/net.lua

```diff
@@ -0,0 +1,30 @@
+local M = {}
+
+function M.download(url, opts)
```

Comment:
I think the `location` should be a mandatory argument; I don't see a "put this _here_ (wherever 'here' is)" is a reasonable use case.